core/src/main/java/org/testcontainers/dockerclient/AuditLoggingDockerClient.java

```diff
@@ -90,7 +90,7 @@ public RemoveNetworkCmd removeNetworkCmd(@NotNull String networkId) {
                                                              BiConsumer<T, Exception> failureConsumer) {
 
         return (T) Proxy.newProxyInstance(
-                clazz.getClassLoader(),
+                Thread.currentThread().getContextClassLoader(),
```

Comment:
this should not be changed, otherwise it will break. Overall, not every `getClassLoader()` should be changed to `getContextClassLoader()`